apps/storybook-react-native/tailwind-intellisense.config.js

```diff
@@ -4,9 +4,7 @@ const designSystemPreset = require('@metamask/design-system-tailwind-preset');
 module.exports = {
   presets: [designSystemPreset],
   content: [
-    '../../packages/design-system-react/src/**/*.{js,jsx,ts,tsx}',
-    '../../packages/design-tokens/stories/**/*.{js,jsx,ts,tsx,mdx}',
-    './src/**/*.{js,jsx,ts,tsx}',
+    '../../packages/design-system-react-native/src/**/*.{js,jsx,ts,tsx}',
```

Comment:
Not that we are using this config file for anything so it probably doesn't matter but we would generally point it to the files that we want to generate the tailwind classnames from